api/models.py

```diff
@@ -185,3 +185,29 @@ class SecurityUser(User):
 
     class Meta:
         verbose_name = "Security User"
+
+
+class AssetLog(models.Model):
+    """Stores checkin/Checkout asset logs"""
+    CHECKIN = "Checkin"
+    CHECKOUT = "Checkout"
+
+    option = (
+        (CHECKIN, "Checkin"),
+        (CHECKOUT, "Checkout"),
+    )
+    item = models.ForeignKey(Item,
```

Comment:
Since going forward we want to use `asset` instead of `item` (see #156704243). Consider changing this to `asset = models.ForeignKey(Item...)`

Reply:
@alexkiura since there's a high priority PT task for refactoring all item references to asset, can I continue using items for now since tests won't pass

If this task can be completed early I pick the refactoring task immediately thereafter